e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -53,6 +53,9 @@ test.describe('Example Imagery Object', () => {
     // Verify that the created object is focused
     await expect(page.locator('.l-browse-bar__object-name')).toContainText(exampleImagery.name);
     await page.getByLabel('Focused Image Element').hover({ trial: true });
+
+    // Wait for image thumbnail auto-scroll to complete
+    await expect(page.getByLabel('Image Thumbnail from').last()).toBeInViewport();
```

Comment:
GOT YOU!!

![gremlins5](https://github.com/user-attachments/assets/7d250628-1008-446a-b07c-fa248906c298)

Reply:
here lies `Can use reset button...` test flake (2022-2024) 🪦